tests/kafkatest/services/kafka/kafka.py

```diff
@@ -410,6 +410,56 @@ def leader(self, topic, partition=0):
         self.logger.info("Leader for topic %s and partition %d is now: %d" % (topic, partition, leader_idx))
         return self.get_node(leader_idx)
 
+    def list_consumer_groups(self, node=None, new_consumer=False, command_config=None):
+        """ Get list of consumer groups.
+        """
+        if node is None:
```

Comment:
Sure I can do that.

> Only thing I'm not sure about making generic is the way you're processing the output (which is the unfortunate result of not having something like JSON output for these commands)

True, Kafka-313 would have addressed that, but it never got traction.